bootstrap/standalone/src/main/java/org/geysermc/platform/standalone/Util.java

```diff
@@ -0,0 +1,44 @@
+package org.geysermc.platform.standalone;
+
+import java.io.InputStream;
+import java.nio.file.Files;
+import java.nio.file.OpenOption;
+import java.nio.file.Paths;
+
+import org.geysermc.platform.standalone.console.GeyserLogger;
+
+public class Util {
```

Comment:
Naming is very vague here, maybe something like `LoopbackUtil` or `LoopbackUtils`.